spec/dummy/db/schema.rb

```diff
@@ -61,6 +48,19 @@
     t.index ["ancestry"], name: "index_feedbook_employees_on_ancestry"
   end
 
+  create_table "feedbook_employees_sessions", force: :cascade do |t|
```

Comment:
Ah, better squash this commit with the one that adds it, it makes no sense to modify a table in this way in the same pull request